src/video-token-server.js

```diff
@@ -5,18 +5,32 @@ const AccessToken = Twilio.jwt.AccessToken;
 const VideoGrant = AccessToken.VideoGrant;
 const MAX_ALLOWED_SESSION_DURATION = 14400;
 
+const ROOM_WHITELIST_PASSCODES = {
```

Comment:
so these passwords are for each user? can a room not have one password that everyone uses?

Reply:
or are these the rooms themselves?

Reply:
The passwords are for each room. It's a coincidence that I made a bunch of default rooms named after folks on the With team.